client/py/aou_workbench_client/cohorts.py

```diff
@@ -0,0 +1,12 @@
+from auth import get_authenticated_swagger_client
+from config import all_of_us_config
+from aou_workbench_client.swagger_client.apis.cohorts_api import CohortsApi
+
+def materialize_cohort(request):
+    """Materializes a cohort in the workspace containing this notebook, based
```

Comment:
Eventually we might want a convenience method that allows a researcher to materialise a cohort without having to create an explicit request object. But that is most definitely a follow on API design activity :-)